testing/acceptance_test.py

```diff
@@ -1326,6 +1326,115 @@ def test_2():
         assert c1 == c2
 
 
+class TestGroupScope:
+    def test_by_module(self, testdir):
```

Comment:
Those tests look great! Could you also add another test which uses 2 different groups (`@pytest.mark.xgroup(name="group1")` and `@pytest.mark.xgroup(name="group2")` for example)?

Reply:
Thank you. 62e50d0